server/src-lib/Hasura/Server/Logging.hs

```diff
@@ -146,6 +146,7 @@ data OperationLog
   , olResponseSize       :: !(Maybe Int64)
   , olQueryExecutionTime :: !(Maybe Double)
   , olQuery              :: !(Maybe Value)
+  , olRawQuery           :: !(Maybe Value)
   , olError              :: !(Maybe Value)
```

Comment:
I think these probably shouldn’t all just be stored as `Value`, since the derived `ToJSON` instance will handle the conversion to JSON for us, so we should store more specific types if we can. I think the reason we’re using `Value` for `olQuery` is that queries have many different types, so we convert them to `Value` beforehand to normalize them, but `olRawQuery` and `olError` can probably have the more specific types `Maybe ByteString` and `Maybe QErr`, respectively.

Reply:
Can `ByteString` be `Text`, because type `ByteString` doesn't have ToJSON instance?